bookmarklet.js

```diff
@@ -31,6 +32,18 @@ javascript:(
                 document.body.removeChild(dummy);
             };
 
+            const downloadFile = (text) => {
+            	if (document.getElementById("download-checkbox").checked){
+                    let dataStr = "data:text/json;charset=utf-8," + encodeURIComponent(text);                
+                    let downloadAnchorNode = document.createElement('a');
+                    downloadAnchorNode.setAttribute("href",     dataStr);
+                    downloadAnchorNode.setAttribute("download",  "download.json");
```

Comment:
```suggestion
                    const dataStr = "data:text/json;charset=utf-8," + encodeURIComponent(text);                
                    const downloadAnchorNode = document.createElement('a');
                    downloadAnchorNode.setAttribute("href", dataStr);
                    downloadAnchorNode.setAttribute("download", "download.json");
```
also if you got this from StackOverflow, could you put a comment where you got it from? 🙏